fluent-dom/src/overlay.js

```diff
@@ -263,19 +267,15 @@ function isAttrNameLocalizable(name, element, explicitlyAllowed = null) {
 }
 
 /**
- * Remove and return the first child of the given type.
+ * Helper to set textContent and localizable attributes on an element.
  *
- * @param {DOMFragment} element
- * @param {string}      localName
- * @returns {Element | null}
+ * @param   {Element} element
+ * @param   {Element} fromElement
+ * @returns {Element}
  * @private
  */
-function shiftNamedElement(element, localName) {
-  for (const child of element.children) {
-    if (child.localName === localName) {
-      element.removeChild(child);
-      return child;
-    }
-  }
-  return null;
+function shallowPopulate(element, fromElement) {
```

Comment:
Can you revert the order of the args here to be consistent with the other methods? I had trouble following assertions of what's safe and what's not because this methods flips that going in and going out to overlayAttributes.